spec/controllers/api/v0/business_days_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Api::V0::BusinessDaysController do
+  describe 'show' do
+    it 'should return a date N business days from the date and time in the DC time zone' do
+      get :show, {date: '2016-08-30', time: '13:00', day_count: '1'}
```

Comment:
i think our ruby style guide, while it does not evaluate specs, calls for spaces within curly braces